Templates/WebHook-GitHub/index.js

```diff
@@ -1,4 +1,8 @@
 ﻿module.exports = function (body, context) {
     context.log('GitHub WebHook triggered! ' + body.comment.body);
-    context.done(null, 'New GitHub comment: ' + body.comment.body);
+    context.done(null, {
+        res: {
+            body: 'New GitHub comment: ' + JSON.stringify(body.comment.body, null, " ")
```

Comment:
It's a bit cleaner, but still only transitional. Once we assume Hook output is JSON, it naturally becomes much simpler.